### PR TITLE
Codename: Au Amber

### PR DESCRIPTION
**Pull request by bderusha**
We are renaming the root of this project to a codename: Au Amber (pronounced: Aye You Amber).

More info: [List of Types of Prunes](http://www.forfruittrees.com/plum-prune.htm)

### CODE REVIEW

**Comment by clintonb** [comment]
Weren't you supposed to rename a single app? Now I care, because I don't like the name. :)

File: au_amber/settings/base.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 path.append(root('apps'))
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.environ.get('COURSE_DISCOVERY_SECRET_KEY', 'insecure-secret-key')
+SECRET_KEY = os.environ.get('au_amber_SECRET_KEY', 'insecure-secret-key')
```
Comment (bderusha) [inline au_amber/settings/base.py:13]: embiggen

File: au_amber/settings/production.py [hunk 2]
```diff
@@ -34,7 +34,7 @@
     )
 }
 
-CONFIG_FILE = get_env_setting('COURSE_DISCOVERY_CFG')
+CONFIG_FILE = get_env_setting('au_amber_CFG')
```
Comment (bderusha) [inline au_amber/settings/production.py:37]: embiggen

File: docker-compose.yml [hunk 3]
```diff
@@ -28,12 +28,12 @@ course-discovery:
...
   # Uncomment the next two lines to build from a local configuration repo
   #build: ../configuration
-  #dockerfile: docker/build/course_discovery/Dockerfile
+  #dockerfile: docker/build/au_amber/Dockerfile
 
   container_name: course-discovery
   volumes:
-    - .:/edx/app/course_discovery/course_discovery
-  command: /edx/app/course_discovery/devstack.sh start
+    - .:/edx/app/au_amber/au_amber
```
Comment (bderusha) [inline docker-compose.yml:35]: Does this also need to change in the config repo?

Comment (clintonb) [inline docker-compose.yml:35]: Yes...which is why I'm inclined to not make this change.

File: Makefile [hunk 4]
```diff
@@ -83,4 +83,4 @@ start-devstack:
 
 open-devstack:
 	docker-compose --x-networking up -d
-	docker exec -it course-discovery env TERM=$(TERM) /edx/app/au_amber/devstack.sh open
+	docker exec -it au-amber env TERM=$(TERM) /edx/app/au_amber/devstack.sh open
```
Comment (bderusha) [inline Makefile:86]: What is this really saying here?

File: docker-compose.yml [hunk 5]
```diff
@@ -22,15 +22,15 @@ memcache:
 #  image: rabbitmq:3.5.3
 #  container_name: rabbitmq
 
-course-discovery:
-  # Uncomment this line to use the official course-discovery base image
-  image: edxops/course-discovery:latest
+au-amber:
+  # Uncomment this line to use the official au-amber base image
```
Comment (bderusha) [inline docker-compose.yml:26]: what work does this represent to change the name of the base image?

File: README.rst [hunk 6]
```diff
@@ -38,7 +38,7 @@ entry point, as it defines the APIs that this service will support.
 Documentation
 -------------
 
-The docs for Course Discovery Service will be on Read the Docs:  https://open-edx-course-discovery.readthedocs.org.
+The docs for Course Metadata Service will be on Read the Docs:  https://open-edx-course-discovery.readthedocs.org.
```
Comment (bderusha) [inline README.rst:41]: What should this be?

`https://open-edx-course-discovery.readthedocs.org`?
`https://open-edx-au-amber.readthedocs.org`?
`https://open-edx-course-metadata.readthedocs.org`?

File: docs/conf.py [hunk 7]
```diff
@@ -48,7 +48,7 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'Course Discovery Service'
+project = u'codename: Au Amber'
```
Comment (bderusha) [inline docs/conf.py:51]: Course Metadata Service. Codename: Au Amber.

**Pull request closed without merging**